composer.json

```diff
@@ -1,5 +1,6 @@
 {
   "require": {
+    "aura/di": "^3.0"
```

Comment:
Warning: I used Aura.Di 2.x; you're using 3.x which has significant changes. Make sure you're aware of that!